### PR TITLE
enh: add the timer ticket creator example

### PR DESCRIPTION
**Pull request by navneel99**
## Summary

Adding the example snap-in for a ticket creator that creates tickets after every 10 minutes.

## Work-item
/ISS-70827

### CODE REVIEW

File: 6-timer-ticket-creator/code/src/fixtures/on_work_created_event.json
Comment (altafDevRev) [inline 6-timer-ticket-creator/code/src/fixtures/on_work_created_event.json:1]: Please rename this as the on work created event might be confusing.

File: 6-timer-ticket-creator/code/src/functions/ticket_creator/index.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    // Create a ticket. Name the ticket using the current date and time.
+    const date = new Date();
+    const ticketName = `Ticket created at ${date.toLocaleString()}`;
+    const ticketBody = `This ticket was created by a snap-in at ${date.toLocaleString()}`;
+
+    const reponse = await devrevSDK.worksCreate({
+      title: ticketName,
+      body: ticketBody,
+      // The ticket will be created in the PROD-1 part. Rename this to match your part.
```
Comment (altafDevRev) [inline 6-timer-ticket-creator/code/src/functions/ticket_creator/index.ts:19]: nit: 
```suggestion
      // The ticket will be created in the PROD-4 part. Rename this to match your part.
```

**Review by altafDevRev — APPROVED** [review]

File: 6-timer-ticket-creator/code/src/fixtures/on_work_created_event.json
Comment (navneel99) [reply 6-timer-ticket-creator/code/src/fixtures/on_work_created_event.json:1]: Updated

File: 6-timer-ticket-creator/code/src/functions/ticket_creator/index.ts (hunk 1 above)
Comment (navneel99) [reply 6-timer-ticket-creator/code/src/functions/ticket_creator/index.ts:19]: Updated the code to use PROD-1. PROD-4 is uncommon

**Pull request merged**